mp-migration-guide/pos.md

```diff
@@ -51,7 +51,7 @@ To make the transition from MobilePay PoS to the ePayment API with Merchant Call
 
 * Host an API with an endpoint that Vipps MobilePay can send callbacks to.
 * Migrate existing QRs. There exists an endpoint specifically for migrating MobilePay QRs. This endpoint will take the `beaconId` and `posName` of the current QR, and register it as a Merchant Callback QR. This will make the old QRs work after the switch to the common platform. The endpoint is: [Create callback QR code](https://developer.vippsmobilepay.com/api/qr/#tag/Merchant-callback-QR/operation/PutMerchantCallbackMobilePayQr).
-* Subscribe to the `user.checkin.v1` event for all merchant serial numbers (i.e., stores). This will trigger Vipps MobilePay to send these events whenever a customer has scanned one of your QRs. The callback contains the `merchant-serial-number`; as well as, the `merchantQrId` (equal to the `beaconId` in MobilePay PoS). This information will distill down to what checkout the customer has checked in to. See more information [here](https://developer.vippsmobilepay.com/docs/APIs/qr-api/vipps-qr-api-webhooks/)
+* Subscribe to the `user.checked-in.v1` event for all merchant serial numbers (i.e., stores). This will trigger Vipps MobilePay to send these events whenever a customer has scanned one of your QRs. The callback contains the `merchant-serial-number`; as well as, the `merchantQrId` (equal to the `beaconId` in MobilePay PoS). This information will distill down to what checkout the customer has checked in to. See more information [here](https://developer.vippsmobilepay.com/docs/APIs/qr-api/vipps-qr-api-webhooks/)
```

Comment:
```suggestion
* Subscribe to the `user.checked-in.v1` event for all merchant serial numbers (i.e., stores). This will trigger Vipps MobilePay to send these events whenever a customer has scanned one of your QRs. The callback contains the `merchant-serial-number`; as well as, the `merchantQrId` (equal to the `beaconId` in MobilePay PoS). This information will distill down to what checkout the customer has checked in to. See more information in the [QR API Webhooks section](https://developer.vippsmobilepay.com/docs/APIs/qr-api/vipps-qr-api-webhooks/)
```